### PR TITLE
Allow sending multiple roles to UNLOAD and COPY commands

### PR DESCRIPTION
**Pull request by alexcarruthers**
## Todos
- [x] MIT compatible
- [x] Tests
- [x] Documentation
- [x] Updated CHANGES.rst

### CODE REVIEW

File: sqlalchemy_redshift/commands.py [hunk 1]
```diff
@@ -28,15 +28,18 @@
 TOKEN_RE = re.compile('[A-Za-z0-9/+=]+')
 AWS_PARTITIONS = frozenset({'aws', 'aws-cn', 'aws-us-gov'})
 AWS_ACCOUNT_ID_RE = re.compile('[0-9]{12}')
-IAM_ROLE_NAME_RE = re.compile('[A-Za-z0-9+=,.@-_]{1,64}')
+IAM_ROLE_NAME_RE = re.compile('[A-Za-z0-9+=,.@\-_]{1,64}')
+IAM_ROLE_ARN_RE = re.compile('arn:(aws|aws-cn|aws-us-gov):iam::[0-9]{12}:role/[A-Za-z0-9+=,.@\-_]{1,64}')
```
[the remark this answers is not in the document]
Comment (alexcarruthers) [reply sqlalchemy_redshift/commands.py:32]: i mostly just went with what this library already had (AWS_ACCOUNT_ID_RE and IAM_ROLE_NAME_RE) and how the role ARN was created. 

I also fixed the accidental range in IAM_ROLE_NAME_RE.

File: sqlalchemy_redshift/commands.py [hunk 2]
```diff
@@ -28,15 +28,18 @@
...
 def _process_aws_credentials(access_key_id=None, secret_access_key=None,
                              session_token=None, aws_partition='aws',
-                             aws_account_id=None, iam_role_name=None):
+                             aws_account_id=None, iam_role_name=None,
+                             iam_role_arns=None):
 
     if (access_key_id is not None and secret_access_key is not None and
-            aws_account_id is not None and iam_role_name is not None):
+            aws_account_id is not None and iam_role_name is not None and
+            iam_role_arns is not None):
```
[the remark this answers is not in the document]
Comment (alexcarruthers) [reply sqlalchemy_redshift/commands.py:42]: that's very true.

File: sqlalchemy_redshift/commands.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexcarruthers) [reply sqlalchemy_redshift/commands.py]: There's also already a comment a few lines above with the link to the documentation.

**Review by jklukas — APPROVED** [review]

**Comment by jklukas** [comment]
Looks good 👍

**Comment by alexcarruthers** [comment]
@jklukas any plans for a release soon?

**Comment by jklukas** [comment]
This change is now released as 0.8.2

**Pull request merged**